added WAM information (#3659)
MicrosoftDocs/azure-docs-cli: docs-ref-conceptual/authenticate-azure-cli.md

To learn more about managed identities for Azure resources, see [Configure managed identities for Azure resources](/azure/active-directory/managed-identities-azure-resources/qs-configure-cli-windows-vm) and [Use managed identities for Azure resources for sign in](/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-sign-in).

> ## Sign in with WAM

Reviewer: ```suggestion
## Sign in with Web Account Manager (WAM)
```

The Azure CLI now offers preview support for Web Account Manager (WAM).  WAM is a Windows 10+ component that acts as an authentication broker.  Using WAM has several benefits:

_ Enhanced security. See [Conditional Access: Token protection (preview)](/azure/active-directory/conditional-access/concept-token-protection).

> _ Support for Windows Hello, conditional access, and FIDO keys.

Reviewer: ```suggestion
_ Support for Windows Hello, conditional access policies, and FIDO keys.
```

> _ Fast single sign-on.

Reviewer: "Fast" seems too subjective. I'd make this more about the platform SSO benefits. 

```suggestion
_ Streamlined single sign-on.
```

_ Bug fixes and enhancements shipped with Windows.

> Signing in with WAM is a preview feature. Once turned on, it will replace the previous browser-based user interface. 

Reviewer: ```suggestion
Signing in with WAM is a preview, opt-in feature. Once enabled, it will replace the previous browser-based user interface. 
```

```azurecli-interactive

> az config set core-allow_broker=true

Reviewer: ```suggestion
az config set core.allow_broker=true
```
